code/FileAttachmentFieldTrack.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Track files as they're uploaded and remove when they've been saved.
+ *
+ * @package  unclecheese/silverstripe-dropzone
+ */
+class FileAttachmentFieldTrack extends DataObject {
+    private static $db = array(
+        'SessionID' => 'Varchar(255)',
```

Comment:
Additional debugging information. Original idea was to try and clean up old sessions, but that proved to be... really dirty: 
http://stackoverflow.com/questions/675913/looping-through-all-a-servers-sessions-in-php

So yeah, this can be omitted.